### ISSUE TITLE
OIP 17 Get a large water filter pitcher and stop ordering still water in bottles

### ISSUE THREAD

**Issue by renaynay** [ethdevberlin/OIPS#17]
It would be nice to have a large water filter pitcher that we can all communally fill up when it is low to filter the tap water (which is a bit hard). We can then slowly stop ordering still/flat bottled water.

**Comment by ligi** [comment]
100% agree
Maybe even replace the bottled carbonated water by getting e.g. a soda-stream

**Comment by leonardoalt** [comment]
I agree with getting rid of bottle water, but why not just drink the tap water? That's at least what I do when I want to drink still water.

**Comment by renaynay** [comment]
Yeah I also drink the tap water, but it's a bit hard in general, so getting a filter would be nice too.

**Comment by franzihei** [comment]
I would also drink the tap water, I'm just missing some kind of carafe to put it in. In FullNode, they had glass carafes to put the water in which was quite convenient.

**Comment by ligi** [comment]
You can use the ones in the big conference room - I was just using them as watering can's for the plants as they where completely unused since we moved to the new office.

**Comment by ligi** [comment]
can this be closed as we now have a water pitcher / filter? Anyone needs a carbonator?

**Comment by leonardoalt** [comment]
I don't think it can be closed since bottled water is still being ordered :p

**Comment by ligi** [comment]
true - leaving it open
